api/swagger/server/restapi/resource/swagger.yml

```diff
@@ -618,6 +618,70 @@ paths:
           description: Internal Server Error - The server has encountered a situation it does not know how to handle.
           schema:
             $ref: "#/definitions/Error"
+  /cmd/deploySC:
+    put:
+      operationId: cmdDeploySC
+      consumes:
+        - multipart/form-data
+      parameters:
+        - in: formData
+          name: nickname
+          type: string
+          required: true
+          x-nullable: false
+          description: Name of the Wallet in which the website will be deployed.
```

Comment:
Is it a SC or a website?
Is the website deploy to site wallet or using the coin of this wallet?